bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ExpressionThreadPoolManager.java

```diff
@@ -86,7 +84,8 @@ public ExpressionThreadPoolExecutor(final String poolName, int corePoolSize) {
                 @Override
                 public void rejectedExecution(Runnable runnable, ThreadPoolExecutor threadPoolExecutor) {
                     // Log and discard
-                    logger.warn("Thread pool '{}' rejected execution of {}",
+                    LoggerFactory.getLogger(ExpressionThreadPoolExecutor.class).warn(
+                            "Thread pool '{}' rejected execution of {}",
                             new Object[] { poolName, runnable.getClass() });
```

Comment:
Not introduced by your changes, but do we need to create an array of the two logger arguments?